grammar/grammar.y

```diff
@@ -161,6 +161,8 @@ WhileTypeExpr <- WhilePrefix TypeExpr (KEYWORD_else Payload? TypeExpr)?
 
 SwitchExpr <- KEYWORD_switch LPAREN Expr RPAREN LBRACE SwitchProngList RBRACE
 
+RangeExpr <- Expr DOT2 Expr?
```

Comment:
This is similar to `SwitchItem` so it would be nice if it was defined similarly.

Reply:
ah, good catch, I'll do that. I might add a for-specific block after the switch specific block as it has similarly unique syntax

Reply:
ok, this is sorted in the latest commit

Reply:
It's better but now it allows omitting the comma between the arguments.